juicy-ace-editor.html

```diff
@@ -113,6 +113,8 @@
                 session.setUseSoftTabs( this.getAttribute("softtabs") );
                 this.getAttribute("tabsize") && session.setTabSize( this.getAttribute("tabsize") );
                 session.setUseWrapMode( this.hasAttribute("wrapmode") );
+                // non ace specific
+                this.hasAttribute("shadow-style") && this.injectTheme(this.getAttribute('shadow-style'));
```

Comment:
I suggest adding support for `shadowStyle` prop too. If you agree I think it should support `HTMLStyleElement`s and `string` CSS Selectors. This gives so much more freedom for the developer.

Reply:
We don't support any other properties than `value` and for YAGNI, I'd leave it that way, unless somebody with real use case asks for it.